tests/test_click_odoo.py

```diff
@@ -235,6 +235,23 @@ def testcmd(env):
     assert "No database provided" in result.output
 
 
+def test_env_options_addons_path():
```

Comment:
So here, start checking script5 fails when addons-path is not set

Reply:
done